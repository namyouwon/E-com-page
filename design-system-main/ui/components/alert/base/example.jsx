// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import Alert from '../';
import { UtilityIcon } from '../../icons/base/example';
import EmptyLink from '../../../shared/empty-link/';
import Heading from '../../../shared/heading/';

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <Alert>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="user"
        />
        <Heading level="2">
          Logged in as John Smith (johnsmith@acme.com).{' '}
          <EmptyLink>Log out</EmptyLink>
        </Heading>
      </Alert>
    )
  }
];

export let states = [
  {
    id: 'theme-info',
    label: 'Default (Theme Class  - Deprecated)',
    element: (
      <Alert isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="user"
        />
        <Heading level="2">
          Logged in as John Smith (johnsmith@acme.com).{' '}
          <EmptyLink>Log out</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'warning',
    label: 'Warning',
    element: (
      <Alert type="warning">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="warning"
        />
        <Heading level="2">
          Your browser is outdated. Your Salesforce experience may be degraded.{' '}
          <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-warning',
    label: 'Warning (Theme Class  - Deprecated)',
    element: (
      <Alert type="warning" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="warning"
        />
        <Heading level="2">
          Your browser is outdated. Your Salesforce experience may be degraded.{' '}
          <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'error',
    label: 'Error',
    element: (
      <Alert type="error">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="error"
        />
        <Heading level="2">
          Your browser is currently not supported. Your Salesforce may be
          degraded. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-error',
    label: 'Error (Theme Class  - Deprecated)',
    element: (
      <Alert type="error" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="error"
        />
        <Heading level="2">
          Your browser is currently not supported. Your Salesforce may be
          degraded. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'offline',
    label: 'Offline',
    element: (
      <Alert type="offline">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="offline"
        />
        <Heading level="2">
          You are in offline mode. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-offline',
    label: 'Offline (Theme Class  - Deprecated)',
    element: (
      <Alert type="offline" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="offline"
        />
        <Heading level="2">
          You are in offline mode. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  }
];
