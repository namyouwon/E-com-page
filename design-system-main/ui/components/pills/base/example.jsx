// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import SvgIcon from '../../../shared/svg-icon';
import { Pill, PillContainer } from '../';

/// ////////////////////////////////////////
// Export
/// ////////////////////////////////////////

export default <Pill />;

export let states = [
  {
    id: 'error',
    label: 'Error',
    element: (
      <Pill className="slds-has-error">
        <span className="slds-pill__icon_container">
          <span className="slds-icon_container" title="Error">
            <SvgIcon
              className="slds-icon slds-icon-text-error"
              sprite="utility"
              symbol="error"
            />
            <span className="slds-assistive-text">Warning</span>
          </span>
        </span>
      </Pill>
    )
  },
  {
    id: 'truncated',
    label: 'Truncated',
    demoStyles: 'width: 220px;',
    storybookStyles: true,
    element: (
      <PillContainer>
        <Pill label="Pill label that is longer than the area that contains it" />
      </PillContainer>
    )
  }
];

export let examples = [
  {
    id: 'icon',
    label: 'With icon',
    element: (
      <Pill>
        <span className="slds-pill__icon_container">
          <span
            className="slds-icon_container slds-icon-standard-account"
            title="Account"
          >
            <SvgIcon className="slds-icon" sprite="standard" symbol="account" />
            <span className="slds-assistive-text">Account</span>
          </span>
        </span>
      </Pill>
    )
  },
  {
    id: 'avatar',
    label: 'With avatar',
    element: (
      <Pill>
        <span className="slds-pill__icon_container">
          <span className="slds-avatar slds-avatar_circle" title="User avatar">
            <img
              alt="Person name"
              src="/assets/images/avatar2.jpg"
              title="User avatar"
            />
          </span>
        </span>
      </Pill>
    )
  },
  {
    id: 'container',
    label: 'Pill with Container',
    element: (
      <PillContainer>
        <Pill />
        <Pill />
        <Pill />
      </PillContainer>
    )
  },
  {
    id: 'container-bare',
    label: 'Pill with Bare Container - Deprecated',
    element: (
      <PillContainer variant="container-bare">
        <Pill />
        <Pill />
        <Pill />
      </PillContainer>
    )
  }
];
