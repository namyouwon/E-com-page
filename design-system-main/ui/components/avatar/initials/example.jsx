// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import { Avatar } from '../base/example';
import { AvatarInitials } from '../';

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <Avatar>
        <AvatarInitials
          iconClassName="slds-icon-standard-account"
          title="company name"
          initials="Ac"
        />
      </Avatar>
    )
  }
];

export let examples = [
  {
    id: 'linked-avatar-initials',
    label: 'Wrapped in a link',
    element: (
      <a href="#" onClick={e => e.preventDefault()}>
        <Avatar>
          <AvatarInitials
            iconClassName="slds-icon-standard-account"
            title="company name"
            initials="Ac"
          />
        </Avatar>
      </a>
    )
  }
];
