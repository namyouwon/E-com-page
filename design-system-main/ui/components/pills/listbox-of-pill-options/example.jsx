// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import {
  PillContainer,
  ListboxPill,
  ListboxPills,
  ListboxPillsItem
} from '../';
import { Avatar } from '../../avatar/base/example';
import { StandardIcon } from '../../icons/standard/example';

/// ////////////////////////////////////////
// Export
/// ////////////////////////////////////////

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <PillContainer type="listbox" id="listbox-pill-default">
        <ListboxPills id="listbox-pill-default">
          <ListboxPillsItem>
            <ListboxPill tabIndex="0" />
          </ListboxPillsItem>
          <ListboxPillsItem>
            <ListboxPill />
          </ListboxPillsItem>
        </ListboxPills>
      </PillContainer>
    )
  }
];

export let examples = [
  {
    id: 'listbox-pill-with-icon',
    label: 'With icon',
    element: (
      <PillContainer type="listbox" id="listbox-pill-with-icon">
        <ListboxPills id="listbox-pill-with-icon">
          <ListboxPillsItem>
            <ListboxPill tabIndex="0">
              <StandardIcon
                containerClassName="slds-pill__icon_container"
                title="Account"
                assistiveText="Account"
              />
            </ListboxPill>
          </ListboxPillsItem>
          <ListboxPillsItem>
            <ListboxPill>
              <StandardIcon
                containerClassName="slds-pill__icon_container"
                symbol="case"
                title="Case"
                assistiveText="Case"
              />
            </ListboxPill>
          </ListboxPillsItem>
        </ListboxPills>
      </PillContainer>
    )
  },
  {
    id: 'listbox-pill-with-avatar',
    label: 'With avatar',
    element: (
      <PillContainer type="listbox" id="listbox-pill-with-avatar">
        <ListboxPills id="listbox-pill-with-avatar">
          <ListboxPillsItem>
            <ListboxPill tabIndex="0">
              <Avatar className="slds-avatar_x-small slds-pill__icon_container">
                <img
                  alt="Person name"
                  src="/assets/images/avatar2.jpg"
                  title="User avatar"
                />
              </Avatar>
            </ListboxPill>
          </ListboxPillsItem>
          <ListboxPillsItem>
            <ListboxPill>
              <Avatar className="slds-avatar_x-small slds-pill__icon_container">
                <img
                  alt="Person name"
                  src="/assets/images/avatar2.jpg"
                  title="User avatar"
                />
              </Avatar>
            </ListboxPill>
          </ListboxPillsItem>
        </ListboxPills>
      </PillContainer>
    )
  },
  {
    id: 'listbox-pill-bare',
    label: 'Bare',
    element: (
      <PillContainer type="listbox" id="listbox-pills-bare">
        <ListboxPills id="listbox-pills-bare">
          <ListboxPillsItem>
            <ListboxPill className="slds-pill_bare" tabIndex="0" />
          </ListboxPillsItem>
          <ListboxPillsItem>
            <ListboxPill className="slds-pill_bare" />
          </ListboxPillsItem>
        </ListboxPills>
      </PillContainer>
    )
  },
  {
    id: 'listbox-pill-group',
    label: 'Grouped',
    element: (
      <div>
        <div className="slds-combobox_container" />
        <PillContainer variant="listbox-group">
          <ListboxPills>
            <ListboxPillsItem>
              <ListboxPill tabIndex="0" />
            </ListboxPillsItem>
            <ListboxPillsItem>
              <ListboxPill />
            </ListboxPillsItem>
          </ListboxPills>
        </PillContainer>
      </div>
    )
  }
];
