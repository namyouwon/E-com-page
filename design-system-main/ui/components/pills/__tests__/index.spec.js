// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
/* eslint-env jest */

import createHelpers from '../../../../jest.helpers';
import { getDisplayElementById } from '../../../shared/helpers';
import * as Pill from '../base/example';
import * as ListboxOfPillOptions from '../listbox-of-pill-options/example';


const { matchesMarkup } = createHelpers(__dirname);

describe('Pill', () => {
  it('renders a pill', () => matchesMarkup(Pill.default));

  it('renders a pill with an icon', () =>
    matchesMarkup(getDisplayElementById(Pill.examples, 'icon')));

  it('renders a pill with an avatar', () =>
    matchesMarkup(getDisplayElementById(Pill.examples, 'avatar')));

  it('renders a pill with an error', () =>
    matchesMarkup(getDisplayElementById(Pill.states, 'error')));

  it('renders a pill that is truncated', () =>
    matchesMarkup(getDisplayElementById(Pill.states, 'truncated')));

  it('renders a pill in a container', () =>
    matchesMarkup(getDisplayElementById(Pill.examples, 'container')));

  it('renders a pill in a container - bare (deprecated)', () =>
    matchesMarkup(getDisplayElementById(Pill.examples, 'container-bare')));
});

describe('Pill - Listbox of Pills', () => {
  it('renders a default listbox of pills', () => matchesMarkup(ListboxOfPillOptions.default));

  it('renders a listbox of pills with an icon', () =>
    matchesMarkup(getDisplayElementById(ListboxOfPillOptions.examples, 'listbox-pill-with-icon')));

  it('renders a listbox of pills with an avatar', () =>
    matchesMarkup(getDisplayElementById(ListboxOfPillOptions.examples, 'listbox-pill-with-avatar')));

  it('renders a listbox of pills - bare', () =>
    matchesMarkup(getDisplayElementById(ListboxOfPillOptions.examples, 'listbox-pill-bare')));

    it('renders a listbox of pills - group', () =>
      matchesMarkup(getDisplayElementById(ListboxOfPillOptions.examples, 'listbox-pill-group')));
});
