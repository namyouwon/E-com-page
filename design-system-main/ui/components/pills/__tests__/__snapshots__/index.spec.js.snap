// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pill - Listbox of Pills renders a default listbox of pills 1`] = `
<div
  className="slds-pill_container"
>
  <div
    className="slds-assistive-text"
    id="listbox-pill-default"
  >
    Press delete or backspace to remove
  </div>
  <ul
    aria-describedby="listbox-pill-default"
    aria-label="Selected Options:"
    aria-orientation="horizontal"
    className="slds-listbox slds-listbox_horizontal"
    role="listbox"
  >
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill"
        role="option"
        tabIndex="0"
      >
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill"
        role="option"
      >
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
  </ul>
</div>
`;

exports[`Pill - Listbox of Pills renders a listbox of pills - bare 1`] = `
<div
  className="slds-pill_container"
>
  <div
    className="slds-assistive-text"
    id="listbox-pills-bare"
  >
    Press delete or backspace to remove
  </div>
  <ul
    aria-describedby="listbox-pills-bare"
    aria-label="Selected Options:"
    aria-orientation="horizontal"
    className="slds-listbox slds-listbox_horizontal"
    role="listbox"
  >
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill slds-pill_bare"
        role="option"
        tabIndex="0"
      >
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill slds-pill_bare"
        role="option"
      >
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
  </ul>
</div>
`;

exports[`Pill - Listbox of Pills renders a listbox of pills - group 1`] = `
<div>
  <div
    className="slds-combobox_container"
  />
  <div
    className="slds-listbox_selection-group"
  >
    <ul
      aria-label="Selected Options:"
      aria-orientation="horizontal"
      className="slds-listbox slds-listbox_horizontal"
      role="listbox"
    >
      <li
        className="slds-listbox-item"
        role="presentation"
      >
        <span
          aria-selected="true"
          className="slds-pill"
          role="option"
          tabIndex="0"
        >
          <span
            className="slds-pill__label"
            title="Full pill label verbiage mirrored here"
          >
            Pill Label
          </span>
          <span
            className="slds-icon_container slds-pill__remove"
            title="Remove"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_x-small slds-icon-text-default"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
              />
            </svg>
          </span>
        </span>
      </li>
      <li
        className="slds-listbox-item"
        role="presentation"
      >
        <span
          aria-selected="true"
          className="slds-pill"
          role="option"
        >
          <span
            className="slds-pill__label"
            title="Full pill label verbiage mirrored here"
          >
            Pill Label
          </span>
          <span
            className="slds-icon_container slds-pill__remove"
            title="Remove"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_x-small slds-icon-text-default"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
              />
            </svg>
          </span>
        </span>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Pill - Listbox of Pills renders a listbox of pills with an avatar 1`] = `
<div
  className="slds-pill_container"
>
  <div
    className="slds-assistive-text"
    id="listbox-pill-with-avatar"
  >
    Press delete or backspace to remove
  </div>
  <ul
    aria-describedby="listbox-pill-with-avatar"
    aria-label="Selected Options:"
    aria-orientation="horizontal"
    className="slds-listbox slds-listbox_horizontal"
    role="listbox"
  >
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill"
        role="option"
        tabIndex="0"
      >
        <span
          className="slds-avatar slds-avatar_x-small slds-pill__icon_container"
        >
          <img
            alt="Person name"
            src="/assets/images/avatar2.jpg"
            title="User avatar"
          />
        </span>
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill"
        role="option"
      >
        <span
          className="slds-avatar slds-avatar_x-small slds-pill__icon_container"
        >
          <img
            alt="Person name"
            src="/assets/images/avatar2.jpg"
            title="User avatar"
          />
        </span>
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
  </ul>
</div>
`;

exports[`Pill - Listbox of Pills renders a listbox of pills with an icon 1`] = `
<div
  className="slds-pill_container"
>
  <div
    className="slds-assistive-text"
    id="listbox-pill-with-icon"
  >
    Press delete or backspace to remove
  </div>
  <ul
    aria-describedby="listbox-pill-with-icon"
    aria-label="Selected Options:"
    aria-orientation="horizontal"
    className="slds-listbox slds-listbox_horizontal"
    role="listbox"
  >
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill"
        role="option"
        tabIndex="0"
      >
        <span
          className="slds-icon_container slds-icon-standard-account slds-pill__icon_container"
          title="Account"
        >
          <svg
            aria-hidden={true}
            className="slds-icon"
          >
            <use
              xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#account"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Account
          </span>
        </span>
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
    <li
      className="slds-listbox-item"
      role="presentation"
    >
      <span
        aria-selected="true"
        className="slds-pill"
        role="option"
      >
        <span
          className="slds-icon_container slds-icon-standard-case slds-pill__icon_container"
          title="Case"
        >
          <svg
            aria-hidden={true}
            className="slds-icon"
          >
            <use
              xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#case"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Case
          </span>
        </span>
        <span
          className="slds-pill__label"
          title="Full pill label verbiage mirrored here"
        >
          Pill Label
        </span>
        <span
          className="slds-icon_container slds-pill__remove"
          title="Remove"
        >
          <svg
            aria-hidden={true}
            className="slds-icon slds-icon_x-small slds-icon-text-default"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
        </span>
      </span>
    </li>
  </ul>
</div>
`;

exports[`Pill renders a pill 1`] = `
<span
  className="slds-pill slds-pill_link"
>
  <a
    className="slds-pill__action"
    href="#"
    onClick={[Function]}
    title="Full pill label verbiage mirrored here"
  >
    <span
      className="slds-pill__label"
    >
      Pill Label
    </span>
  </a>
  <button
    className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
    onClick={[Function]}
    title="Remove"
  >
    <svg
      aria-hidden={true}
      className="slds-button__icon"
    >
      <use
        xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
      />
    </svg>
    <span
      className="slds-assistive-text"
    >
      Remove
    </span>
  </button>
</span>
`;

exports[`Pill renders a pill in a container - bare (deprecated) 1`] = `
<div
  className="slds-pill_container slds-pill_container_bare"
>
  <span
    className="slds-pill slds-pill_link"
  >
    <a
      className="slds-pill__action"
      href="#"
      onClick={[Function]}
      title="Full pill label verbiage mirrored here"
    >
      <span
        className="slds-pill__label"
      >
        Pill Label
      </span>
    </a>
    <button
      className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
      onClick={[Function]}
      title="Remove"
    >
      <svg
        aria-hidden={true}
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Remove
      </span>
    </button>
  </span>
  <span
    className="slds-pill slds-pill_link"
  >
    <a
      className="slds-pill__action"
      href="#"
      onClick={[Function]}
      title="Full pill label verbiage mirrored here"
    >
      <span
        className="slds-pill__label"
      >
        Pill Label
      </span>
    </a>
    <button
      className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
      onClick={[Function]}
      title="Remove"
    >
      <svg
        aria-hidden={true}
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Remove
      </span>
    </button>
  </span>
  <span
    className="slds-pill slds-pill_link"
  >
    <a
      className="slds-pill__action"
      href="#"
      onClick={[Function]}
      title="Full pill label verbiage mirrored here"
    >
      <span
        className="slds-pill__label"
      >
        Pill Label
      </span>
    </a>
    <button
      className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
      onClick={[Function]}
      title="Remove"
    >
      <svg
        aria-hidden={true}
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Remove
      </span>
    </button>
  </span>
</div>
`;

exports[`Pill renders a pill in a container 1`] = `
<div
  className="slds-pill_container"
>
  <span
    className="slds-pill slds-pill_link"
  >
    <a
      className="slds-pill__action"
      href="#"
      onClick={[Function]}
      title="Full pill label verbiage mirrored here"
    >
      <span
        className="slds-pill__label"
      >
        Pill Label
      </span>
    </a>
    <button
      className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
      onClick={[Function]}
      title="Remove"
    >
      <svg
        aria-hidden={true}
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Remove
      </span>
    </button>
  </span>
  <span
    className="slds-pill slds-pill_link"
  >
    <a
      className="slds-pill__action"
      href="#"
      onClick={[Function]}
      title="Full pill label verbiage mirrored here"
    >
      <span
        className="slds-pill__label"
      >
        Pill Label
      </span>
    </a>
    <button
      className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
      onClick={[Function]}
      title="Remove"
    >
      <svg
        aria-hidden={true}
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Remove
      </span>
    </button>
  </span>
  <span
    className="slds-pill slds-pill_link"
  >
    <a
      className="slds-pill__action"
      href="#"
      onClick={[Function]}
      title="Full pill label verbiage mirrored here"
    >
      <span
        className="slds-pill__label"
      >
        Pill Label
      </span>
    </a>
    <button
      className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
      onClick={[Function]}
      title="Remove"
    >
      <svg
        aria-hidden={true}
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Remove
      </span>
    </button>
  </span>
</div>
`;

exports[`Pill renders a pill that is truncated 1`] = `
<div
  className="slds-pill_container"
>
  <span
    className="slds-pill slds-pill_link"
  >
    <a
      className="slds-pill__action"
      href="#"
      onClick={[Function]}
      title="Pill label that is longer than the area that contains it"
    >
      <span
        className="slds-pill__label"
      >
        Pill label that is longer than the area that contains it
      </span>
    </a>
    <button
      className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
      onClick={[Function]}
      title="Remove"
    >
      <svg
        aria-hidden={true}
        className="slds-button__icon"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Remove
      </span>
    </button>
  </span>
</div>
`;

exports[`Pill renders a pill with an avatar 1`] = `
<span
  className="slds-pill slds-pill_link"
>
  <span
    className="slds-pill__icon_container"
  >
    <span
      className="slds-avatar slds-avatar_circle"
      title="User avatar"
    >
      <img
        alt="Person name"
        src="/assets/images/avatar2.jpg"
        title="User avatar"
      />
    </span>
  </span>
  <a
    className="slds-pill__action"
    href="#"
    onClick={[Function]}
    title="Full pill label verbiage mirrored here"
  >
    <span
      className="slds-pill__label"
    >
      Pill Label
    </span>
  </a>
  <button
    className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
    onClick={[Function]}
    title="Remove"
  >
    <svg
      aria-hidden={true}
      className="slds-button__icon"
    >
      <use
        xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
      />
    </svg>
    <span
      className="slds-assistive-text"
    >
      Remove
    </span>
  </button>
</span>
`;

exports[`Pill renders a pill with an error 1`] = `
<span
  className="slds-pill slds-pill_link slds-has-error"
>
  <span
    className="slds-pill__icon_container"
  >
    <span
      className="slds-icon_container"
      title="Error"
    >
      <svg
        aria-hidden={true}
        className="slds-icon slds-icon-text-error"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#error"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Warning
      </span>
    </span>
  </span>
  <a
    className="slds-pill__action"
    href="#"
    onClick={[Function]}
    title="Full pill label verbiage mirrored here"
  >
    <span
      className="slds-pill__label"
    >
      Pill Label
    </span>
  </a>
  <button
    className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
    onClick={[Function]}
    title="Remove"
  >
    <svg
      aria-hidden={true}
      className="slds-button__icon"
    >
      <use
        xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
      />
    </svg>
    <span
      className="slds-assistive-text"
    >
      Remove
    </span>
  </button>
</span>
`;

exports[`Pill renders a pill with an icon 1`] = `
<span
  className="slds-pill slds-pill_link"
>
  <span
    className="slds-pill__icon_container"
  >
    <span
      className="slds-icon_container slds-icon-standard-account"
      title="Account"
    >
      <svg
        aria-hidden={true}
        className="slds-icon"
      >
        <use
          xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#account"
        />
      </svg>
      <span
        className="slds-assistive-text"
      >
        Account
      </span>
    </span>
  </span>
  <a
    className="slds-pill__action"
    href="#"
    onClick={[Function]}
    title="Full pill label verbiage mirrored here"
  >
    <span
      className="slds-pill__label"
    >
      Pill Label
    </span>
  </a>
  <button
    className="slds-button slds-button_icon slds-button_icon slds-pill__remove"
    onClick={[Function]}
    title="Remove"
  >
    <svg
      aria-hidden={true}
      className="slds-button__icon"
    >
      <use
        xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
      />
    </svg>
    <span
      className="slds-assistive-text"
    >
      Remove
    </span>
  </button>
</span>
`;
